packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -414,10 +414,11 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
   Future<bool> isMarkerInfoWindowShown(
     MarkerId markerId, {
     required int mapId,
-  }) {
+  }) async {
     assert(markerId != null);
-    return channel(mapId).invokeMethod<bool>('markers#isInfoWindowShown',
-        <String, String>{'markerId': markerId.value}) as Future<bool>;
+    bool? res  = await channel(mapId).invokeMethod<bool?>('markers#isInfoWindowShown',
```

Comment:
Why are you changing this to `<bool?>`? The signature is `Future<T?> invokeMethod <T>`, so I would not expect this to need an optional type.

Reply:
You're right again, I confused T with the Future<T?> return type.

This should be the final code:
```
  return (await channel(mapId).invokeMethod<bool>('markers#isInfoWindowShown',
        <String, String>{'markerId': markerId.value}))! ;
```